core-ui/src/shared/components/EventSubscriptionConditionStatus.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { StatusBadge } from 'react-shared';
+
+export const EventSubscriptionConditionStatus = ({ condition, isListView }) => {
+  const { t, i18n } = useTranslation();
+  const statusBadgeProperties =
+    condition?.status === 'True'
+      ? {
+          type: 'success',
+          text: t('common.statuses.ready'),
+          tooltipMessage: condition?.type,
+        }
+      : {
+          type: 'error',
+          text: t('common.statuses.error'),
+          tooltipMessage: condition?.message,
+        };
+  return (
+    <StatusBadge
+      type={statusBadgeProperties.type}
+      additionalContent={statusBadgeProperties.tooltipMessage}
+      i18n={i18n}
+      noTooltip={condition?.status !== 'False' && !isListView}
```

Comment:
Why don't we show the ready status for the details view?

Reply:
I asked @Wawrzyn321 what he thought about it and he said that there was no need to inform a user twice because the `Type` information was already there (e.g. 'Subscription Active'). We can discuss it with other team members.

Reply:
Referring to your first comment, I'll add deep links for functions and services.